tool/tctl/common/plugin/awsic.go

```diff
@@ -0,0 +1,242 @@
+	}
+
+	if !a.useSystemCredentials {
+		return trace.BadParameter("only AWS Local system credentials are supported")
+	}
+
+	return nil
+}
+
+// parseAWSICNameFilters validates that all elements of the supplied [names] slice
+// are valid regexes or globs and wraps them in [types.AWSICResourceFilter]s for
+// inclusion in a [types.PluginAWSICSettings].
+//
+// We are using a manual validator here rather than the canonical one defined
+// in the AWS IC integration itself, because those filter tools are not
+// available to OSS builds of tctl.
+//
+// TODO(tcsc): Move the filter validation somewhere that the that the OSS build can access it.
+func parseAWSICNameFilters(names []string) ([]*types.AWSICResourceFilter, error) {
+	var filters []*types.AWSICResourceFilter
+	for _, n := range names {
+		if _, err := utils.CompileExpression(n); err != nil {
+			return nil, trace.Wrap(err)
+		}
+		filters = append(filters, &types.AWSICResourceFilter{
+			Include: &types.AWSICResourceFilter_NameRegex{NameRegex: n},
+		})
+	}
+	return filters, nil
+}
+
+func (a *awsICArgs) parseGroupFilters() ([]*types.AWSICResourceFilter, error) {
+	return parseAWSICNameFilters(a.groupNameFilters)
+}
+
+func (a *awsICArgs) parseAccountFilters() ([]*types.AWSICResourceFilter, error) {
+	filters, err := parseAWSICNameFilters(a.accountNameFilters)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	for _, id := range a.accountIDFilters {
+		filters = append(filters, &types.AWSICResourceFilter{
+			Include: &types.AWSICResourceFilter_Id{Id: id},
+		})
+	}
+
+	return filters, nil
+}
+
+func (a *awsICArgs) parseUserFilters() ([]*types.AWSICUserSyncFilter, error) {
+	var result []*types.AWSICUserSyncFilter
+
+	if len(a.userOrigins) > 0 {
+		result = make([]*types.AWSICUserSyncFilter, 0, len(a.userOrigins))
+		for _, origin := range a.userOrigins {
+			result = append(result, &types.AWSICUserSyncFilter{
+				Labels: map[string]string{types.OriginLabel: origin},
+			})
+		}
+	}
+
+	if len(a.userLabels) > 0 {
+		result = append(make([]*types.AWSICUserSyncFilter, 0, len(result)+len(a.userLabels)), result...)
+		for _, labels := range a.userLabels {
+			result = append(result, &types.AWSICUserSyncFilter{Labels: labels})
+		}
+	}
+
+	return result, nil
+}
+
+func (p *PluginsCommand) initInstallAWSIC(parent *kingpin.CmdClause) {
+	p.install.awsIC.cmd = parent.Command("awsic", "Install an AWS IAM Identity Center integration.")
+	cmd := p.install.awsIC.cmd
+	cmd.Flag("access-list-default-owner", "Teleport user to set as default owner for the imported access lists. Multiple flags allowed.").
+		Required().
+		StringsVar(&p.install.awsIC.defaultOwners)
+	cmd.Flag("scim-url", "AWS Identity Center SCIM provisioning endpoint").
+		Required().
+		URLVar(&p.install.awsIC.scimURL)
+	cmd.Flag("scim-token", "AWS Identify Center SCIM provisioning token.").
+		Required().
+		StringVar(&p.install.awsIC.scimToken)
+	cmd.Flag("instance-region", "AWS Identity center instance region").
```

Comment:
```suggestion
	cmd.Flag("instance-region", "AWS Identity Center instance region").
```

Here and below, let's consistently capitalize Identity Center.